lib/fileio/commands/baseExportCommand.h

```diff
@@ -0,0 +1,109 @@
+//
+// Copyright 2016 Pixar
+// Copyright 2020 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef MAYA_EXPORT_COMMAND_H
+#define MAYA_EXPORT_COMMAND_H
+
+#include "../../base/api.h"
```

Comment:
Yeah, I'll make this change.  In general I avoided making too many changes related to #includes here because of the on-going conversation about them and that we were probably going to batch edit everything in one big clean up step.  This one was copied from somewhere else I think, but it's the one style change that we all agree should be changed so I might as well do it now.